Model/PaymentMethod.php

```diff
@@ -218,4 +236,28 @@ public function getSupportConfigFields()
 
         return $aFieldsPublic;
     }
+
+    /**
+     * Returns the post-processing payment method for this payment method
+     *
+     * @return PaymentMethod
+     *
+     * @since 1.0.1
+     */
+    public function getPostProcessingPaymentMethod()
```

Comment:
is it not just a get instance? i don't see the connection why post processing payment method.

Reply:
You just need difference transactions that all as the only setter needed is the parent transaction one.

Reply:
we need the paymentMethod to configure the transaction and get the correct post processing buttons